saleor/plugins/webhook/plugin.py

```diff
@@ -45,6 +47,32 @@ def order_updated(self, order: "Order", previous_value: Any) -> Any:
         order_data = generate_order_payload(order)
         trigger_webhooks_for_event.delay(WebhookEventType.ORDER_UPDATED, order_data)
 
+    def invoice_request(
+        self,
+        order: "Order",
+        invoice: "Invoice",
+        number: Optional[str],
+        previous_value: Any,
+    ) -> Any:
+        if not self.active:
+            return previous_value
+        invoice_data = generate_invoice_payload(invoice)
```

Comment:
Do you want to send `invoice` object here? AFAIK this method will be called when staff user wants to generate an Invoice, which means that this object will be empty. Shouldn't we send here an order data?

Reply:
invoice might get in handy since that contains job id. And yes, will add order inside.